### PR TITLE
Add boltdb hook

### PR DESCRIPTION
**Pull request by taras-skycoin**
Did you run `make format && make check`?
yes
Fixes inconsistent logging in terminal
 Changes:	
-	Add BoltDB logger hook

How to test this PR:
Just run skywire-visor

### CODE REVIEW

File: pkg/app/appcommon/log_store.go [hunk 1]
```diff
@@ -4,16 +4,21 @@ import (
...
 	"github.com/skycoin/skycoin/src/util/logging"
 	"go.etcd.io/bbolt"
 )
 
-const timeLayout = time.RFC3339Nano
+const (
+	timeLayout = time.RFC3339Nano
+)
+
+var re = regexp.MustCompile(`\x1b?\[[0-9;]*m`)
```
Comment (nkryuchkov) [inline pkg/app/appcommon/log_store.go]: Could you please use any library instead? For example, this one: https://github.com/acarl005/stripansi. The regular expression there could cover more cases.

File: pkg/app/appcommon/log_store.go [hunk 2]
```diff
@@ -24,7 +29,8 @@ func NewProcLogger(conf ProcConfig) (*logging.MasterLogger, LogStore) {
 
 	log := logging.NewMasterLogger()
 	log.Logger.Formatter.(*logging.TextFormatter).TimestampFormat = time.RFC3339Nano
-	log.SetOutput(io.MultiWriter(os.Stdout, db))
+	// log.SetOutput(io.MultiWriter(os.Stdout, db))
```
Comment (nkryuchkov) [inline pkg/app/appcommon/log_store.go]: If this line is not needed, could you please remove it?

File: pkg/app/appcommon/log_store.go [hunk 3]
```diff
@@ -24,7 +29,8 @@ func NewProcLogger(conf ProcConfig) (*logging.MasterLogger, LogStore) {
 
 	log := logging.NewMasterLogger()
 	log.Logger.Formatter.(*logging.TextFormatter).TimestampFormat = time.RFC3339Nano
```
Comment (nkryuchkov) [inline pkg/app/appcommon/log_store.go]: It's not a part of your PR, but I think we could change `time.RFC3339Nano` to `timeLayout` here.

**Review by nkryuchkov — COMMENTED** [review]
Logs look fine. I will review again when logs of VPN apps are changed.

File: pkg/app/appcommon/log_store.go (hunk 1 above)
Comment (taras-skycoin) [reply pkg/app/appcommon/log_store.go]: @nkryuchkov I think it better just copy their regular expression.

File: cmd/apps/vpn-client/vpn-client.go [hunk 4]
```diff
@@ -49,25 +50,29 @@ func main() {
...
 		//*serverPKStr = "03e9019b3caa021dbee1c23e6295c6034ab4623aec50802fcfdd19764568e2958d"
-		log.Fatalln("VPN server pub key is missing")
+		fmt.Println("VPN server pub key is missing")
+		os.Exit(1)
 	}
 
 	serverPK := cipher.PubKey{}
 	if err := serverPK.UnmarshalText([]byte(*serverPKStr)); err != nil {
-		log.WithError(err).Fatalln("Invalid VPN server pub key")
+		fmt.Printf("Invalid VPN server pub key. error = %v", err)
```
Comment (nkryuchkov) [inline cmd/apps/vpn-client/vpn-client.go]: Could you please change the message to `Invalid VPN server pub key: %v` and apply the same format to other log messages? The format is `fmt.Printf("Error log message: %v", err)`.

File: internal/vpn/client.go [hunk 5]
```diff
@@ -151,9 +151,9 @@ func (c *Client) Serve() error {
 		return fmt.Errorf("error during client/server handshake: %w", err)
 	}
 
-	c.log.Infof("Performed handshake with %s", c.conn.RemoteAddr())
-	c.log.Infof("Local TUN IP: %s", tunIP.String())
-	c.log.Infof("Local TUN gateway: %s", tunGateway.String())
+	fmt.Printf("Performed handshake with %s", c.conn.RemoteAddr())
```
Comment (nkryuchkov) [inline internal/vpn/client.go:152]: Is `c.log` not used anymore? If so, could you please remove it from `Client`?

File: internal/vpn/client.go [hunk 6]
```diff
@@ -214,14 +214,14 @@ func (c *Client) Serve() error {
 		defer close(connToTunDoneCh)
 
 		if _, err := io.Copy(tun, c.conn); err != nil {
-			c.log.WithError(err).Errorf("Error resending traffic from TUN %s to VPN server", tun.Name())
+		fmt.Printf("Error resending traffic from TUN %s to VPN server. error = %v", tun.Name(), err))
```
Comment (nkryuchkov) [inline internal/vpn/client.go]: This needs to be formatted. Could you please run `goimports` for this file? There's also an extra bracket at the end of the line.

File: internal/vpn/client.go [hunk 7]
```diff
@@ -214,14 +214,14 @@ func (c *Client) Serve() error {
...
 		}
 	}()
 	go func() {
 		defer close(tunToConnCh)
 
 		if _, err := io.Copy(c.conn, tun); err != nil {
-			c.log.WithError(err).Errorf("Error resending traffic from VPN server to TUN %s", tun.Name())
+		fmt.Printf("Error resending traffic from VPN server to TUN %s. error = %v", tun.Name(), err)
```
Comment (nkryuchkov) [inline internal/vpn/client.go]: Needs to be formatted too.

File: internal/vpn/client.go [hunk 8]
```diff
@@ -255,14 +255,14 @@ func (c *Client) routeTrafficThroughTUN(tunGateway net.IP) error {
 }
 
 func (c *Client) routeTrafficDirectly(tunGateway net.IP) {
-	c.log.Infoln("Routing all traffic through default network gateway")
+fmt.Println("Routing all traffic through default network gateway")
```
Comment (nkryuchkov) [inline internal/vpn/client.go]: Needs to be formatted.

**Review by nkryuchkov — CHANGES_REQUESTED** [review]

File: cmd/apps/skysocks/skysocks.go [hunk 9]
```diff
@@ -56,11 +57,17 @@ func main() {
 		<-termCh
 
 		if err := srv.Close(); err != nil {
-			log.Fatalf("Failed to close server: %v", err)
+			if err != nil {
```
Comment (nkryuchkov) [inline cmd/apps/skysocks/skysocks.go]: Could you please remove the extra error check?

File: cmd/apps/skysocks/skysocks.go [hunk 10]
```diff
@@ -56,11 +57,17 @@ func main() {
...
+				fmt.Println(err)
+				os.Exit(1)
+			}
 		}
 	}()
 
 	if err := srv.Serve(l); err != nil {
-		log.Fatal(err)
+		if err != nil {
```
Comment (nkryuchkov) [inline cmd/apps/skysocks/skysocks.go]: Extra error check.

File: cmd/apps/vpn-client/vpn-client.go [hunk 11]
```diff
@@ -49,25 +50,29 @@ func main() {
...
+		os.Exit(1)
 	}
 
 	localPK := cipher.PubKey{}
 	if *localPKStr != "" {
 		if err := localPK.UnmarshalText([]byte(*localPKStr)); err != nil {
-			log.WithError(err).Fatalln("Invalid local PK")
+			fmt.Printf("Invalid local PK. error = %v", err)
```
Comment (nkryuchkov) [inline cmd/apps/vpn-client/vpn-client.go]: Could you please change the output format here and for the rest of the file?

**Review by nkryuchkov — CHANGES_REQUESTED** [review]

**Pull request merged**